packages/core/src/utils/zod.ts

```diff
@@ -34,14 +41,8 @@ export const zodTypeToSwagger = (config: unknown): OpenAPIV3.SchemaObject => {
     };
   }
 
-  if (config instanceof ZodUnion) {
-    return {
-      description: 'union',
-    };
-  }
-
   if (config instanceof ZodUnknown) {
-    return { description: 'unknown' };
+    return {};
```

Comment:
better leave a comment here. Technically, it is not the original ZodUnknown use case.

Reply:
Updated. PTAL

```ts
  if (config instanceof ZodUnknown) {
    // Since there is no `unknown` data type in OpenAPI schemas, use `any` instead.
    // `{}` is shorthand syntax for an arbitrary-type schema, and we can add a description on it.
    // OpenAPI Data Types: https://swagger.io/docs/specification/data-models/data-types
    return { description: 'The specific type depends.' };
  }
```

In redoc:

<img width="416" alt="image" src="https://user-images.githubusercontent.com/10594507/172100322-6308b7ae-59a4-44ce-8e97-ec06365295aa.png">